src/java/org/apache/cassandra/service/paxos/cleanup/PaxosCleanupLocalCoordinator.java

```diff
@@ -58,6 +61,18 @@ public class PaxosCleanupLocalCoordinator extends AsyncFuture<PaxosCleanupRespon
     private static final Logger logger = LoggerFactory.getLogger(PaxosCleanupLocalCoordinator.class);
     private static final UUID INTERNAL_SESSION = new UUID(0, 0);
 
+    private static class DelayedRepair
+    {
+        private final UncommittedPaxosKey uncommitted;
+        private final long startAfterMillis;
```

Comment:
I assumed this was a time delta when I read it (i.e. after millis (have elapsed)). Perhaps `atMillis` or `startAtMillis`, or `scheduledAtMillis`?

But also, this should probably be nanos and we should probably use `nanoTime`?